tests/epyccel/test_arrays.py

```diff
@@ -1495,6 +1511,1171 @@ def test_array_variable_size():
     s1, s2 = f2(n,m)
     assert s1 == s2
 
+#==============================================================================
+# TEST : 1d array slices
+#==============================================================================
+
+def test_array_1d_slice_1():
+    a = np.empty(21, dtype=np.int)
+    for i in range(21):
+        a[i] = 1 << i
+    f1 = arrays.array_1d_slice_1
+    f2 = epyccel(f1)
+
+    assert np.array_equal(f1(a), f2(a))
+
+def test_array_1d_slice_2():
+    a = np.empty(21, dtype=np.int)
+    for i in range(21):
+        a[i] = 1 << i
```

Comment:
See above list comprehension. There are many more unnecessary for loops in this file